plugins/node-cert/commands/check-inspection.md

```
@@ -0,0 +1,94 @@
+# node-cert check-inspection
+
+Detect if SSL/TLS inspection is active on the current network.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)